src/exchanges.js

```diff
@@ -12,6 +12,13 @@ var amqplib       = require('amqplib');
 var events        = require('events');
 var util          = require('util');
 var common        = require('./common');
+var taskcluster   = require('taskcluster-client');
+
+// wait 30 seconds before closing a channel, to allow pending operations to flush
+var CLOSE_DELAY = 30 * 1000;
+
+// unconditionally reconnect to pulse on this interval
+var RECONNECT_INTERVAL = "30 minutes"
 
```

Comment:
I want to make sure the reconnecting works.  If we configure it so that it never occurs in Heroku, then someday when we run an app *not* in Heroku, we will suddenly find out it's broken.  Reconnection should be a normal, everyday occurrence.

We could raise the interval to six or twelve hours -- it's hard to see a performance impact at that point.  What do you think @jonasfj ?